lib/matplotlib/axes/_base.py

```diff
@@ -589,9 +590,18 @@ def __init__(self, fig, rect,
         fig : `~matplotlib.figure.Figure`
             The Axes is built in the `.Figure` *fig*.
 
-        rect : tuple (left, bottom, width, height).
-            The Axes is built in the rectangle *rect*. *rect* is in
-            `.Figure` coordinates.
+        *args
```

Comment:
Is this needed to unite the Axes and Subplot constuctor APIs? That's quite ugly. We should consider to migrate three numbers to "one tuple of three numbers" so that we have always exactly one argument `rect` back and can save the additional `rect in kwargs` handling.

But that may be for later.

Reply:
Yes, this is super ugly and I do want to move to "one tuple of 3 numbers", but that cannot be done with yet another deprecation cycle which can be done later.